openassessment/xblock/staff_area_mixin.py

```diff
@@ -512,7 +512,7 @@ def clear_team_state(self, user_id, course_id, item_id, requesting_user_id, subm
         )
         # Tell the submissions API to orphan the submissions to prevent them from being accessed
         from submissions import team_api as team_submissions_api
-        team_submissions_api.reset_scores(team_submission_uuid)
+        team_submissions_api.reset_scores(team_submission_uuid, clear_state=True)
```

Comment:
Should there be a test updated to reflect this?

Reply:
The only non-integration way to test it that I can think of just asserts the correct arguments, but I don't know how useful it would be.

Reply:
I would say the bulk of the testing will happen once the integration is complete with the other two prs linked above.

Reply:
I reverse that, @muselesscreator, I did find a way to test it. Good call 👍